src/components/MapView/Analyser/index.tsx

```diff
@@ -39,18 +40,22 @@ const baselineLayers = layers.filter(
 const hazardLayers = layers.filter(
   (layer): layer is WMSLayerProps => layer.type === 'wms',
 );
+const statistics = ['mean', 'median'];
+
 const boundaryLayer = getBoundaryLayerSingleton();
 
 const apiUrl = 'https://prism-api.ovio.org/stats'; // TODO both needs to be stored somewhere
 const adminJson =
   'https://prism-admin-boundaries.s3.us-east-2.amazonaws.com/mng_admin_boundaries.json';
-async function submitAnaysisRequest(
+async function submitAnalysisRequest(
   baselineLayer: NSOLayerProps,
   hazardLayer: WMSLayerProps,
-  extent: Extent,
+  extent: Extent | undefined,
```

Comment:
and on line 55, why are you removing these types?

Reply:
These are temporary and we will fix it in upcoming changes